addons/mail/models/mail_thread.py

```diff
@@ -843,7 +837,17 @@ def _routing_check_route(self, message, message_dict, route, raise_exception=Tru
                 check_result = self.env['mail.alias.mixin']._alias_check_contact_on_record(obj, message, message_dict, alias)
             if check_result is not True:
                 self._routing_warn(_('alias %s: %s') % (alias.alias_name, check_result.get('error_message', _('unknown error'))), message_id, route, False)
-                self._routing_create_bounce_email(email_from, check_result.get('error_template', _generic_bounce_body_html), message)
+                _generic_bounce_body_html = _("""Hello,
```

Comment:
Moar crazy stuff: if we have a partner linked to the incoming email, try to render in this partner language instead of current user (what '_' does). So much crazy, but could be interesting to try.